code/modules/clothing/head/tactical.dm

```diff
@@ -12,6 +12,8 @@
 	name = "tactical helmet"
 	desc = "Standard Security gear. Protects the head from impacts. Can be attached with a flashlight."
 	icon_state = "helmet_sec"
+	species_fit = list(VOX_SHAPED)
```

Comment:
what is going on in here

Reply:
He did put together some new vox sprites, but it should be in its own PR.

Reply:
that shouldn't be a merge issue, since that change was already merged in a previous PR I did